test/intersections.jl

```diff
@@ -529,12 +529,24 @@
     # crossing in one point
     l1 = Line(P3(1,2,3), P3(2,1,0))
     l2 = Line(P3(1,2,3), P3(1,1,1))
-    @test l1 ∩ l2 == l2 ∩ l1 == P3(1,2,3)
+    @test l1 ∩ l2 ≈ l2 ∩ l1 ≈ P3(1,2,3)
 
```

Comment:
Can you also add more tests for the cases where the previous code was failing? I see that you replaced the == by \approx and added a couple @inferred tests. Can we also add tests for the cases that motivated your proposed fix?

Reply:
sure, i will add a test, with varying the one point using a for loop